theme/static/js/custom.js

```diff
@@ -191,7 +176,185 @@ $(document).ready(function () {
             }
         },
         error: showUniversalMessage()
-    })
+    });
+
+    // Event trigger for profile preview
+    $(".profile-preview").click(function () {
+        // Move the profile card below the clicked item
+        var profileCard = $(this).parent().find(".profile-card");
+        profileCard.css("top", ($(this).position().top + 30) + "px");
```

Comment:
I don't do tons in the frontend, but shouldn't this type of thing be done by switching out classes?  I guess that might not be possible with the references to position and width?  I don't know, seems pretty messy.

Reply:
Not in this case because we need to acccess the top() method which is a dynamic value.